lib/morse_translator.rb

```diff
@@ -38,6 +41,31 @@ def initialize
                     "9" => "----.",
                     "0" => "-----",
                     " " => " "}
+    @morse_code_string_array = []
+    @english_array = []
+  end
+
+  def translate(word)
+    downcase = word.downcase
+    array_of_strings = downcase.each_char.to_a
+    array_of_strings.each do |letter|
+      @morse_code_string_array << @dictionary[letter]
```

Comment:
Try to opt for local variables over instance variables. Use your instance variables sparingly. For this example, you could rewrite this method as follows:

```
def translate(word)
  downcase = word.downcase
  array_of_strings = downcase.each_char.to_a
  morse_code_string_array = []
  array_of_strings.each do |letter|
    morse_code_string_array << @dictionary[letter]
  end
  return morse_code_string_array.join
end
```